### PR TITLE
Adopt exposure survey

### PR DESCRIPTION
**Pull request by MonikaFu**
Adopt the exposure plots in survey section

### CODE REVIEW

File: R/prep_exposures_survey.R [hunk 1]
```diff
@@ -18,45 +18,52 @@
 #' @return data.frame
 prep_exposures_survey <- function(results_portfolio,
                                   peers_results_aggregated,
-                                  sector = c("coal", "oil and gas"),
+                                  sector = c("coal", "oil_and_gas"),
```
Comment (jacobvjk) [inline R/prep_exposures_survey.R:21]: I am okay with doing this, but just mentioning that the underscores are inconsistent with how lower case technologies are mapped in P4B. So we end up having different conventions again

**Review by jacobvjk — COMMENTED** [review]
see comments

**Comment by MonikaFu** [comment]
@jacobvjk I believe I did all the changes requested

**Review by jacobvjk — APPROVED** [review]
lgtm

**Pull request merged**